packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/MultipleChoiceTask/MultipleChoiceTask.js

```diff
@@ -54,14 +54,18 @@ class MultipleChoiceTask extends React.Component {
   render () {
     const {
       annotations,
+      disabled,
       task
     } = this.props
     let annotation
     if (annotations && annotations.size > 0) {
       annotation = annotations.get(task.taskKey)
     }
     return (
-      <StyledFieldset autoFocus={(annotation && annotation.value && annotation.value.length === 0)}>
+      <StyledFieldset
+        autoFocus={(annotation && annotation.value && annotation.value.length === 0)}
```

Comment:
Fieldsets aren't focusable elements in HTML, so I suspect this a line that was copied over from PFE even though it does nothing here. It can probably be removed.

Reply:
Yep, it wouldn't just been a straight lift from PFE. Let's open an issue to update the auto focus for the task area.